layouts/partials/docs/nav.html

```diff
@@ -66,22 +128,24 @@
     <div class="nav-sub-section">
       {{ range . }}
       {{ $isHere := eq $here .RelPermalink }}
-      <a class="nav-sub-section-title{{ if $isHere }} is-active{{ end }}" href="{{ .RelPermalink }}">
-        {{ .Title }}
-      </a>
-
-      {{ with .RegularPages }}
-      <ul class="nav-sub-section-pages">
-        {{ range . }}
-        {{ $isHere := eq $here .RelPermalink }}
-        <li class="nav-sub-section-pages-link">
-          <a{{ if $isHere }} class="is-active"{{ end }} href="{{ .RelPermalink }}">
-          {{ .Title }}
-          </a>
-        </li>
+      <details class="nav-section-details"{{ if $isHere }} close{{ end }} {{ if eq (len .RegularPages) 0 }}no-expand{{ end }}>
```

Comment:
Suggest opening the detailed pages if the page is the current page, so that will be apparent for users to check them when navigating to the section page w/o another extra click.

`close` and `no-expand` seems customized attributes, so I am not sure if there would be some side-effects when displaying the page in different browsers? from my experience, probably it should be fine, but also can consider using `data-*` for customized attributes, [ref](https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes).